src/main/resources/sql/load_ts_tables.sql

```diff
@@ -1,59 +1,48 @@
-CREATE TABLE ts_packets (
-	id             serial primary key,
-    project_id     integer not null references projects(id) on delete cascade on update cascade,
-	packet_id      integer not null,
-    plot_id        integer not null references user_plots (id) on delete cascade on update cascade
+DROP TABLE IF EXISTS packets cascade;
+CREATE TABLE packets (
+  packet_id     integer not null,
+  project_rid   integer not null references projects(project_uid) on delete cascade on update cascade,
+  plot_rid      integer not null references plots(plot_uid) on delete cascade on update cascade
 );
-CREATE INDEX packets_pp ON ts_packets USING btree (project_id, packet_id);
+DROP INDEX IF EXISTS packets_pp;
+CREATE INDEX packets_pp ON packets USING btree (project_rid, packet_id);
 
-
-DROP TABLE IF EXISTS ts_project_user cascade;
-CREATE TABLE ts_project_user (
-  id            bigserial primary key,
-  project_id    integer not null references projects(id) on delete cascade on update cascade,
-  packet_id     integer DEFAULT NULL,
-  user_id       integer not null references users (id) on delete cascade on update cascade,
-  date_assigned timestamp NULL DEFAULT CURRENT_TIMESTAMP,
-  isactive      integer DEFAULT 1,
-  complete_date timestamp NULL DEFAULT NULL
-);
-
-DROP TABLE IF EXISTS ts_response_design cascade;
-CREATE TABLE ts_response_design (
-    id       serial primary key,
-    project_id      integer not null references projects(id) on delete cascade on update cascade,
-    landuse         text NOT NULL,
-    landcover       text NOT NULL,
-    change_process  text NOT NULL
-);
-DROP INDEX IF EXISTS prjid;
-CREATE UNIQUE INDEX prjid on ts_response_design (project_id);
-
-DROP TABLE IF EXISTS ts_plot_comments cascade;
-CREATE TABLE ts_plot_comments (
-  id                    bigserial primary key,
-  project_id            integer not null references projects(id) on update cascade,
-  plot_id               integer not null references user_plots (id) on delete cascade on update cascade,
-  interpreter           integer not null references users (id) on update cascade,
+DROP TABLE IF EXISTS plot_comments cascade;
+CREATE TABLE plot_comments (
+  plot_comments_uid           bigserial primary key,
+  project_rid           integer not null references projects(project_uid) on update cascade,
+  plot_rid              integer not null references plots(plot_uid) on delete cascade on update cascade,
+  user_rid              integer not null references users(user_uid) on update cascade,
   comment               text,
   is_example            integer DEFAULT NULL,
   is_complete           integer DEFAULT NULL,
   is_wetland            integer DEFAULT NULL,
   uncertainty           integer DEFAULT NULL,
   last_modified_date    timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  packet_id             integer default -1
+  packet_id             integer default -1 --references packets(packet_id) on delete cascade on update cascade
```

Comment:
if this is going to reference packet_uid, change to packet_rid

Reply:
changed based on yesterday and today's discussion.